infrastructures/kubernetes/src/test/java/org/eclipse/che/workspace/infrastructure/kubernetes/wsnext/KubernetesWorkspaceNextApplierTest.java

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.workspace.infrastructure.kubernetes.wsnext;
+
+import static com.google.common.collect.ImmutableMap.of;
+import static java.util.Collections.emptyList;
+import static java.util.Collections.emptyMap;
+import static java.util.Collections.singletonList;
+import static java.util.Collections.singletonMap;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyZeroInteractions;
+import static org.mockito.Mockito.when;
+import static org.testng.Assert.assertEquals;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import io.fabric8.kubernetes.api.model.Container;
+import io.fabric8.kubernetes.api.model.ObjectMeta;
+import io.fabric8.kubernetes.api.model.Pod;
+import io.fabric8.kubernetes.api.model.PodSpec;
+import io.fabric8.kubernetes.api.model.Quantity;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import org.eclipse.che.api.workspace.server.spi.InfrastructureException;
+import org.eclipse.che.api.workspace.server.spi.environment.InternalMachineConfig;
+import org.eclipse.che.api.workspace.server.wsnext.model.CheService;
+import org.eclipse.che.api.workspace.server.wsnext.model.CheServiceSpec;
+import org.eclipse.che.api.workspace.server.wsnext.model.EnvVar;
+import org.eclipse.che.api.workspace.server.wsnext.model.ResourceRequirements;
+import org.eclipse.che.workspace.infrastructure.kubernetes.environment.KubernetesEnvironment;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.Mock;
+import org.mockito.testng.MockitoTestNGListener;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Listeners;
+import org.testng.annotations.Test;
+
+/** @author Alexander Garagatyi */
+@Listeners(MockitoTestNGListener.class)
+public class KubernetesWorkspaceNextApplierTest {
+  private static final String TEST_IMAGE = "testImage/test:test";
+  private static final String ENV_VAR = "PLUGINS_ENV_VAR";
+  private static final String ENV_VAR_VALUE = "PLUGINS_ENV_VAR_VALUE";
+  private static final String MEMORY_KEY = "memory";
+  private static final String MEMORY_VALUE = "100Mi";
+  private static final String POD_NAME = "pod12";
+  private static final Map<String, String> RESOURCES_REQUEST = ImmutableMap.of(MEMORY_KEY, MEMORY_VALUE);
+
+
+  @Mock
+  Pod pod;
```

Comment:
Looks like unformatted. Please don't forget to execute `mvn fmt:format` before merge.
Also, I would make these variable private. It doesn't really matter while it is test class but still.

Reply:
ok